src/SWP/Component/Common/Generator/GeneratorInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
```

Comment:
'Generator' is a bit of a generic name, I think. What does it generate? Anything? I can't imagine how such a generic interface could be useful.